docs/marketplace/dev/feature-walkthroughs/202108.0/persistence-acl-feature-walkthrough/configuration.md

```diff
@@ -0,0 +1,276 @@
+---
+title: Configuration
+last_updated: Sep 14, 2021
+template: concept-topic-template
+---
+
+Persistence ACL feature based functionality on Propel behavior. It's possible to enable the feature in 2 ways:
+- Connect the feature to one or several tables in the database
+- Connect the feature to all tables in the database
+
+## Connect Persistence ACL feature to one or several tables
+```xml
```

Comment:
Please mention smth like: In the following code snippet only SpyMerchant entity is configured to be handled by ACL.